packages/plexus/README.md

```diff
@@ -1,13 +1,914 @@
+
+This util can be used to hide text when it would be too small to read:
+
+```css
+.demo-graph.is-small .demo-node {
+  color: transparent;
+}
+```
+
+```tsx
+<LayeredDigraph
+  edges={edges}
+  vertices={vertices}
+  setOnGraph={[{ className: 'demo-graph' }, classNameIsSmall]}
+  layoutManager={lm}
+  measurableNodesKey="nodes"
+  layers={[
+    {
+      key: 'nodes',
+      layerType: 'html',
+      measurable: true,
+      // Alternatively, it can be used on the nodes layer
+      // setOnContainer: classNameIsSmall,
+      renderNode: (vertex: TVertex) => vertex.name,
+      setOnNode: { className: 'demo-node' },
+    },
+    {
+      key: 'edges',
+      edges: true,
+      layerType: 'svg',
+      defs: [{ localId: 'arrow' }],
+      markerEndId: 'arrow',
+    },
+  ]}
+/>
+```
+
+Alternatively, it could be set on the `setOnContainer` field of the nodes layer.
+
+#### `scaleProperty.opacity`
+
+This utility will generate a style prop with the opacity reduced as the view zooms out.
+
+In the following example, the opacity of the edges will be reduced as the view is zoomed out.
+
+```tsx
+<LayeredDigraph
+  edges={edges}
+  vertices={vertices}
+  layoutManager={lm}
+  measurableNodesKey="nodes"
+  layers={[
+    {
+      key: 'nodes',
+      layerType: 'html',
+      measurable: true,
+      renderNode: (vertex: TVertex) => vertex.name,
+    },
+    {
+      key: 'edges',
+      edges: true,
+      layerType: 'svg',
+      setOnContainer: scaleProperty.opacity,
+      defs: [{ localId: 'arrow' }],
+      markerEndId: 'arrow',
+    },
+  ]}
+/>
+```
+
+#### `scaleProperty.strokeOpacity`
+
+This is the same as `scaleProperty.opacity` (above) but it reduces the `stroke-opacity` when the view is zoomed out.
+
+#### `scaleProperty`
+
+`scaleProperty` is a factory function for creating utilities like `scaleProperty.opacity` that interpolate the value of a CSS property based on the scale of the graph's zoom transform. For instance, `scaleProperty.opacity` reduces the opacity as the scale of the graph reduces (i.e. as the user zooms out).
+
+The typedef for the factory is:
+
+```tsx
+function scaleProperty(
+    property: keyof React.CSSProperties,
+    valueMin: number = 0.3,
+    valueMax: number = 1,
+    expAdjuster: number = 0.5
+) => (graphState: TExposedGraphState) => React.CSSProperties;
+```
+
+With the default values, the property will approach `0.3` as the scale of the zoom transform approaches `0`. The `expAdjuster` is an exponent applied to the linear change. By default, the interpolation is based on the square root of the linear change.
+
+If you need something more expressive, take a look at `packages/plexus/src/LayeredDigraph/utils.tsx`, which `scaleProperty` wraps.
+
+## Recipes
+
+TODO:
```

Comment:
Are these (or some of these) things we think we'll tackle while doing DDG work? If so, can we add a bullet point to related stories to come back here and update this section?

Reply:
The only catch is, for the border recipes they're already in the demo app, just need to add them to this section.